svg.panzoom.js

```diff
@@ -129,16 +140,18 @@ SVG.extend(SVG.Doc, SVG.Nested, {
       lastP = currentP
     }
 
-    this.on('wheel', zoom)
+    this.on('wheel', wheelZoom)
     this.on('touchstart', pinchZoomStart, this, {passive:false})
     this.on('mousedown', panStart, this, {passive:false})
     this.on('touchstart', panStart, this, {passive:false})
 
+    this.zoom = zoom
```

Comment:
I can do that but then we loose the global `zoomAnimate` option which is not a terrible idea. Since it's only useful in very edge cases, where you want to save a few lines of code to have a default custom animation function for both programmable and mouse-wheel zoom. In most cases, it's probably harmful and could possibly lead to confusion for the developer.